.travis.yml

```diff
@@ -13,7 +13,18 @@ rust:
 before_install:
   - pyenv global system 3.6
 
+install:
```

Comment:
Yeah that would simplify things. If we want a single binary for easy deployment on CoreOS then we can't rely on Nix, because we would need to link against the existing libc and libcurl, not against those in the Nix store. But for CI Nix would be definitely nice. Do you know how to test against multiple rustc versions with Nix? I found https://github.com/mozilla/nixpkgs-mozilla but it looks like it only provides overlays for rustc release channels, not specific versions.

Reply:
You can change the RPaths that nix's linkers sets with Nix's https://nixos.org/patchelf.html  utility .  It is possible to create dynamically linked libs with Nix that can be used outside of Nix with this.